Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -426,6 +428,22 @@ static bool is_header_value(String const& header_value)
     return true;
 }
 
+static Tuple<ByteBuffer, String> safely_extract_body(XMLHttpRequestBodyInit& body)
+{
+    if (body.has<NonnullRefPtr<URL::URLSearchParams>>()) {
+        return {
+            body.get<NonnullRefPtr<URL::URLSearchParams>>()->to_string().to_byte_buffer(),
+            "application/x-www-form-urlencoded;charset=UTF-8"
+        };
+    } else if (body.has<String>()) {
```

Comment:
No else after return, etc etc.
```suggestion
    }
    VERIFY(body.has<String>());
```